playbooks/modules/alert_settings.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Alert Settings Module Operations on PowerScale Storage
+  hosts: localhost
+  connection: local
+  vars:
+    onefs_host: "x.x.x.x"
+    api_user: "user"
+    api_password: "xxxx"
+    verify_ssl: false
+
+  tasks:
+    - name: Get alert settings
```

Comment:
This should be possible only from a info module.

Reply:
Done.